component/cart/cart.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://webkit-ui.netlify.app/style.css">
+    <link rel="stylesheet" href="/css/style.css">
+    <script src="https://kit.fontawesome.com/85de537a36.js" crossorigin="anonymous"></script>
+    <title>Cart</title>
+</head>
+
+<body>
+    <div class="h-100">
+
+
+        <!--- Navigation bar -->
+        <header class=" bg-green-7 text-color-0 pd-5 position-fixed t-0">
```

Comment:
if possible use semantic HTML here.
wrap the whole nav bar in `nav` tag

Reply:
Yes I have actually forgot to add semantic tag.
Thank you for pointing out.